### PR TITLE
Leaflet 1.x

### PR DESCRIPTION
**Pull request by elesdoar**
Solve conflicts on #170  and my wrong PR #172

### CODE REVIEW

**Comment by elesdoar** [comment]
@nmccready please confirm that all is good?

**Comment by nmccready** [comment]
Wow what is going on here? Did my previous merge of master into leaflet-1.X not work? The history on this does not seem right? If this is that messed up starting leaflet-1.X from scratch.. ie a new branch from current master might be better.

**Comment by elesdoar** [comment]
I resolve all conflicts, but did a PR on wrong branch, basically on master branch :disappointed:, by now, this works, but some test not run for leaflet 1.x incompatibility. In as I work on this, I'll go to resolve it.

**Comment by nmccready** [comment]
Got it, the PR I started was targeted the wrong direction; (leaflet-1.X into master)  and then this is to resolve it.

File: package.json [hunk 1]
```diff
@@ -37,35 +43,35 @@
...
     "karmaChrome": "grunt test-unit-chrome",
     "karmaChromeOnce": "grunt karma:unit-chrome-once"
   },
-  "main": "dist/angular-leaflet-directive",
+  "main": "dist/ui-leaflet.js",
   "dependencies": {
     "angular": "1.x",
     "angular-simple-logger": "~0.1.4",
+    "karma-coverage": "^0.5.3",
     "leaflet": "1.0.0-beta.1"
```
Comment (nmccready) [inline package.json]: Just noting the beta.1, fix after merge @elesdoar

**Comment by nmccready** [comment]
Yeah this now makes sense LGTM

**Pull request merged**